_extras/design.md

```diff
@@ -0,0 +1,130 @@
+    *   Learners could be provided with decision tree and solution code for each  exercise
+
+## Stage 1 - Desired Results
+
+### Goals
+
+1.  Learners will be able to navigate [Choosing a Good Chart][good-chart].
+    *   The appropriate graphic should be chosen to respond directly to the scientific question/hypothesis that is to be addressed with the data
+    *   Key decisions
+        1.  What is the structure of the data/results to communicate?
+            *   Relationship
+            *   Comparison
+            *   Distribution
+            *   Composition
+        2.  How many variables or categories need to be conveyed?
+            *   Is one of those variables time?
+        3.  How much data is available for each variable or category?
+        4.  Are absolute or relative values more appropriate to communicate?
+2.  Learners will be able to implement the appropriate graphical device using `dplyr` and `ggplot2`.
+    *   Basic graphing layers
+        *   `ggplot()` 
+        *   `aes()`
+        *   `geom_line()`
+        *   `geom_bar()`
+        *   `geom_smooth()`
+        *   `geom_point()`        
+        *   `geom_boxplot()`
+        *   `geom_smooth()`
+    *   Multiple plots
+        *   `facet_grid()`
+        *   `facet_wrap()`
+    *   Dataset and aesthetic adjustments (*Publication quality figures*)
+        *   `scale_manual()`
+        *   `lims()`
+        *   `labs()`
+        *   `guide_legend()`
+        *   `theme()`        
+
+### Summative Assessment
+
+*   Midpoint: Given a research question and some data, choose the best chart.
+*   Final: Evaluate an analysis report generated from code that includes four or five pairs of research questions and charts:
+    1.  Modify some data that is laid out differently (*but is still tabular*) than what is required by the graphing process
+    2.  Find and fix a chart that is not the best option to address the research question
+    3.  Modify one chart to be publication quality
+
+### Essential Questions
+
+How do I...
+
+*   Read, analyze, and visualize a tabular data set?
+*   Manipulate, summarize, and analyze data to answer a research question?
+*   Choose the best chart to answer my research question?
+*   Generate the best chart to answer my research question?
+*   Produce a publication quality version of my chart?
+
+### Learners Will Be Able To...
+
+*   Interactively generate analysis code and save it in a file
+*   Read tabular data into a data frame
+*   Manipulate and summarize tabular data using `dplyr`
+*   Generate publication quality charts using `ggplot2`
+*   Access `help()`, vignette tutorials, and online documentation
+*   Interpret common error/warning messages from `dplyr` and `ggplot2`
+
+### Learners Will Know...
+
+*   That a clearly defined research question is essential to a successful analysis and graphical representation of the results
+*   That charts should be chosen to best answer the research question
```

Comment:
This one is sort of given: nobody would ever argue that a chart should be chosen to give the *worst* answer to a research question...